Make deepchecks work with classification tasks when no proba is given (#2081)
deepchecks/deepchecks: docs/source/user-guide/general/metrics_guide.rst

Classification
______________
.. Note::

>     For classification tasks, Deepchecks require an ordered

Reviewer: ```suggestion
    For classification tasks, Deepchecks requires an ordered
```

    list of all possible classes (Can also be inferred from provided data and model).

>     It is also recommended to supply the model's output probabilities per class, as some metrics and checks require

Reviewer: ```suggestion
    It is also recommended to supply the model's output probabilities per class, as they are required for some metrics and checks
```

>     that.

Reviewer: ```suggestion
    which will not work without them.
```
